### ISSUE TITLE
Symbol.toPrimitive is not reserved

### ISSUE THREAD

**Issue by blooddy** [google/closure-compiler#3346]
input:
```javascript
class A {
  [ Symbol.toPrimitive ]() {
    return 5;
  }
}
console.log( 5 + new A() );
```
output:
```javascript
class a {
  [Symbol.a]() {
    return 5;
  }
}
console.log(5 + new a);
```

**Comment by tjgq** [comment]
Created Google internal issue b/131172228.

This should be fixed by adding an externs declaration for `Symbol.toPrimitive` in https://github.com/google/closure-compiler/blob/master/externs/es3.js#L70. Would you like to send us a PR?

**Comment by qqingvarqq** [comment]
I found that this issue more general
closure-compiler polyfill "Symbol" and part of well-known symbols are inside es3 definition file -> https://github.com/google/closure-compiler/blob/master/externs/es3.js#L64
Including:
``` Javascript
Symbol.iterator;
Symbol.toStringTag;
Symbol.unscopables;  
/* Missing Symbol.toPrimitive */
```
and es6 definition file -> https://github.com/google/closure-compiler/blob/master/externs/es6.js#L1771
Including only
``` Javascript
Symbol.asyncIterator;
```
Other well-known Symbols  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol#Well-known_symbols are not defined in the es6 definition file, which causes errors when we use ADVANCED_OPTIMIZATION and transpile to  ECMASCRIPT6;
Input:
``` Javascript
console.log(Symbol.hasInstance);
console.log(Symbol.isConcatSpreadable);
console.log(Symbol.species);

/* closure-compiler warn as, that these Property species never defined on Symbol, but not truncate those properties */
console.log(Symbol.match);
console.log(Symbol.matchAll);
console.log(Symbol.replace);
console.log(Symbol.search);
console.log(Symbol.split);
```
Output:
``` Javascript
'use strict';
console.log(Symbol.a);
console.log(Symbol.b);
console.log(Symbol.c);
console.log(Symbol.match);
console.log(Symbol.matchAll);
console.log(Symbol.replace);
console.log(Symbol.search);
console.log(Symbol.split);
```
https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%2540language_out%2520ECMASCRIPT6%250A%252F%252F%2520%2540formatting%2520pretty_print%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250Aconsole.log(Symbol.hasInstance)%253B%250Aconsole.log(Symbol.isConcatSpreadable)%253B%250Aconsole.log(Symbol.species)%253B%250A%250A%252F*%2520closure-compiler%2520warn%2520as%252C%2520that%2520these%2520Property%2520species%2520never%2520defined%2520on%2520Symbol%252C%2520but%2520not%2520truncate%2520those%2520properties%2520*%252F%250Aconsole.log(Symbol.match)%253B%250Aconsole.log(Symbol.matchAll)%253B%250Aconsole.log(Symbol.replace)%253B%250Aconsole.log(Symbol.search)%253B%250Aconsole.log(Symbol.split)%253B
So I assume we should add those Symbols to the es6 definition file.
And I would send PR to this issue if no objections?

**Comment by blickly** [comment]
Right, if they are part of the ES6 spec, es6.js is the right externs file for them.

PR would be great!
